chassis.go

```diff
@@ -123,6 +123,29 @@ func (odbi *ovndb) chassisDelImp(name string) (*OvnCommand, error) {
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
+func (odbi *ovndb) chassisListImp() ([]*Chassis, error) {
+	var listChassis []*Chassis
+
+	odbi.cachemutex.RLock()
+	defer odbi.cachemutex.RUnlock()
+
+	cacheChassis, ok := odbi.cache[tableChassis]
+
+	if !ok {
+		return nil, ErrorSchema
+	}
+
+	for uuid := range cacheChassis {
+		ch, err := odbi.rowToChassis(uuid)
+		if err != nil {
+			return nil, err
+		}
+		listChassis = append(listChassis, ch)
+		fmt.Printf("uuid is %v\n", uuid)
```

Comment:
nit: no need to print uuid.

Reply:
@noah8713: Good catch. That was left over from some testing. I have fixed it now